docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-post-kcp.md

```diff
@@ -60,7 +60,7 @@ containerized workloads from the four resources built into kcp TMC
 are meaningful in KubeStellar.
 
 ```shell
-kubestellar init
+IN_CLUSTER=false SPACE_MANAGER_KUBECONFIG=~/.kube/config kubestellar init
```

Comment:
It looks like some updates are required to https://docs.kubestellar.io/main/Coding%20Milestones/PoC2023q1/commands/#kubestellar-init to describe the newly required environment variables.

Reply:
We will handle this as part of PR3 which continues the work on example1.

Reply:
I think it is better if the documentation is updated simultaneously with the code it documents. Otherwise we spend some time in a confused state. The design here is complicated enough that I am having trouble keeping it in my head. Having it declared in the document would help.